src/components/AddComment.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { postComment } from "../assets/axiosGet";
+import { useState, useContext } from "react";
+import { UsernameContext } from "./UsernameContext";
+
+import "../styles/AddComment.css"
+
+export default function AddComment({article_id}) {
+
+    const [commentData, setCommentData] = useState({username: "", body: ""})
+
+    const {username, setUsername} = useContext(UsernameContext)
+
+    const handleSubmit = (e) =>{
+        e.preventDefault()
+        alert("comment posted!")
+        postComment(article_id, commentData)
+        setCommentData({username: "", body: ""})
+    }
```

Comment:
Why is this set to a blank username and body? This should ideally be updated with the comments and the new comment.